src/ports/postgres/modules/graph/pagerank.py_in

```diff
@@ -30,6 +30,8 @@
 import plpy
 from utilities.control import MinWarning
 from utilities.utilities import _assert
+from utilities.utilities import _check_groups
+from utilities.utilities import _grp_from_table
```

Comment:
Currently, these functions are not used in pagerank, but similar functionalities are implemented. We should refactor those pagerank-customized implementations to use these functions. For instance the variables `group_by_clause` and `grouping_where_clause` can be created using the above functions.